app/assets/styles/partials/components/_input-type.scss

```diff
@@ -73,3 +73,28 @@
     }
   }
 }
+
+.input-type--unit {
```

Comment:
No need to duplicate the CSS from `.input-type--percentage`, just add the selector for `.input-type--unit` to that block instead:

```scss
  .input-type--percentage,
  .input-type--unit {
    // css
  }
```